Hymns/Infrastructure/Data/HymnalApiService.swift

```diff
@@ -6,14 +6,11 @@ import Resolver
  * Service to contact the Hymnal API.
  */
 protocol HymnalApiService {
-    func getHymn(hymnType: HymnType, hymnNumber: String, queryParams: [String: String]?) -> AnyPublisher<Hymn, ErrorType>
+    func getHymn(_ hymnIdentifier: HymnIdentifier) -> AnyPublisher<Hymn, ErrorType>
```

Comment:
Is their an advantage to removing the label using _ ? Just wondering what made you decide to go this route, seems like it would decrease readability across the code base.

Reply:
From looking at how others do it, "_" is used to make things less verbose when it's clear what the argument is. So for example, for getHymn(...), it's clear from the method name that you are trying to get a hymn, so passing in the hymn identifier is intuitive without an explicit label? I dunno. What do you think, for this particular case?

Reply:
Yeah, that makes sense. And I agree it is pretty intuitive here. I was just wondering.